test/unit/operations/get_more.test.ts

```diff
@@ -69,7 +69,7 @@ describe('GetMoreOperation', function () {
         const call = stub.getCall(0);
         expect(call.args[0]).to.equal(namespace);
         expect(call.args[1]).to.deep.equal(expectedGetMoreCommand);
-        expect(call.args[2]).to.deep.equal(opts);
+        expect(call.args[2]).to.containSubset(opts);
```

Comment:
this test was ensuring that the command is called with the expected options, changing this to a subset changes that guarantee - can we update it instead to the expected options?

Reply:
cc: @aditi-khare-mongoDB

Reply:
addressed in https://github.com/mongodb/node-mongodb-native/pull/4315/files